conf/functions_worker.yml

```diff
@@ -57,6 +57,7 @@ pulsarServiceUrl: pulsar://localhost:6650
 # For TLS:
 # webServiceUrl=https://localhost:8443/
 pulsarWebServiceUrl: http://localhost:8080
+functionWebServiceUrl: http://localhost:8080
```

Comment:
I think this will change the behavior of the following code if `workerHostname` is overridden in this file and the operator is relying on the following conditional to set the `functionWebServiceUrl`. Why do we need to add this to the configuration?

https://github.com/apache/pulsar/blob/64bbbd499736825d60a778718b2e7834163d998b/pulsar-functions/worker/src/main/java/org/apache/pulsar/functions/worker/PulsarWorkerService.java#L432-L435

https://github.com/apache/pulsar/blob/64bbbd499736825d60a778718b2e7834163d998b/pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/worker/WorkerConfig.java#L618-L620

https://github.com/apache/pulsar/blob/64bbbd499736825d60a778718b2e7834163d998b/pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/worker/WorkerConfig.java#L599-L604